samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt

```diff
@@ -52,7 +52,7 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdgeSimple()
         setContent {
             AppThemeSurface {
-                SettingsLazyScaffold(
+                SimpleLazyListScaffold(
```

Comment:
it will change to SimpleScaffold soon with the new PR FYI as i think it makes more sense

Reply:
Updated.